database.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
The database connection uses the outdated 'mysqli' extension. It is recommended to use the 'PDO' extension for a more modern and flexible database connection approach. Implement error handling for connection failures to manage them more gracefully.